manage/scheduler.rst

```diff
@@ -82,6 +82,19 @@ Disable notifications
    By default, actions triggered by schedulers won't send notifications.
    You can override this here by setting this to *no*.
 
+Locale
+   Set a desired language, in which the output of variables is
+   translated into. This setting only has an effect if you create an article
+   with variables by the scheduler job. Additionally, the value has to be
+   translated. For more information, please have a look at
```

Comment:
We suggest rephrasing it:

"Set a desired language, in which the output of variables is translated into by using the `t` function. For more information, please have a look at..."